test/units/modules/network/eos/test_eos_l2_interfaces.py

```diff
@@ -80,20 +80,24 @@ def test_eos_l2_interfaces_replaced(self):
         set_module_args(dict(
             config=[dict(
                 name="Ethernet2",
+                mode="trunk",
                 trunk=dict(native_vlan=50)
             ), dict(
                 name="Ethernet3",
                 access=dict(vlan=30),
             )], state="replaced"
         ))
-        commands = ['interface Ethernet2', 'switchport trunk native vlan 50',
-                    'interface Ethernet3', 'switchport access vlan 30']
+        commands = ['interface Ethernet2',
+                    'switchport trunk native vlan 50',
```

Comment:
`switchport mode trunk` needs to be added.

Reply:
since the cfg file already have [switchport mode trunk](https://github.com/ansible/ansible/blob/devel/test/units/modules/network/eos/fixtures/eos_l2_interfaces_config.cfg#L6), the command is not fired.